### DISCUSSION TITLE
van-image 好像不支持 base64 的图片

### DISCUSSION THREAD

**Discussion by v012345** [youzan/vant#10218] [Ideas]
开发过程中,用了一个生成二维码的库,返回的二维码是base64编码的,直接用<img>是可以的,不过<van-image>好像还不支持.

**Comment by qvani** [comment]
是可以的，`<van-image src="data:image/jpeg;base64,你要加的base64码"/> `
这样就可以了，记得看图片格式，image/jpeg改成你要展现的图片对应格式，例如 image/png等等
